runtime/error_test.go

```diff
@@ -483,133 +483,6 @@ func TestRuntimeError(t *testing.T) {
 	})
 }
 
-func TestRuntimeDefaultFunctionConflictPrintingError(t *testing.T) {
```

Comment:
This error is no longer produced. Below test case tests the related other errors.

Reply:
The test seems to have tested the error printing. Maybe there was a problem in the error type before? In that case the test can be removed. Maybe the test was also exercising something in the error pretty printer? (the long body in test function is suspicious)

Reply:
The error pretty printing for related other errors is tested in the test immediately below this one (e.g: `TestRuntimeMultipleInterfaceDefaultImplementationsError`). This one tests the same for `DefaultFunctionConflictError` specifically, which no longer exists. So I think this is safe to remove.

Reply:
cc: @dsainati1

Reply:
Yeah, if the error no longer exists it makes sense to me to remove this test.